WifiManager/Wifi.json

```diff
@@ -337,12 +347,12 @@
             }
         },
         "initiateWPSPairing":{
-            "summary": "Initiates a connection using WPS. A `0` value indicates that paring was initiated. A nonzero value indicates that paring was not initiated.",
+            "summary": "Initiates a connection using Wifi Protected Setup (WPS). An existing connection will be disconnected before attempting to initiate a new connection. If the existing connection cannot be disconnected, then the WPS pairing will stop, and an error event triggers.",
```

Comment:
As mentioned in my first set of review comments (https://github.com/rdkcentral/rdkservices/pull/1904#discussion_r731227120), this part is not true and should be removed:
"If the existing connection cannot be disconnected, then the WPS pairing will stop, and an error event triggers."